content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -15,62 +15,209 @@ further_reading:
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_inspect_details.png" alt="The View device details tab of an individual device with the Interfaces tab selected" style="width:80%;" >}}
 
 ### View flow details
 
 Choose **View flow details** to open the NetFlow tab filtered by the device's `@device.ip` for a detailed exploration of the device's sources, destinations, and volume. See the [NetFlow Monitoring][1] page for more information.
 
-### Troubleshooting
+## Troubleshooting
+
+If you experience issues using the Network Topology Map, use the following troubleshooting guidelines. If you need further assistance, contact [Datadog support][5].
+
+### Missing topology data message box
+
+{{< img src="/network_device_monitoring/network_topology_map/missing_topology_data_2.png" alt="The missing topology data message that is displayed when the rendered map has no links" style="width:80%;" >}}
+
+This message is displayed when the rendered map has no links.
+
+**Note**: Since the "Hide _N_ Unconnected Devices" toggle is enabled by default, this message displays with an empty map. 
 
-If you don't see links or connections to your devices, verify that they are exposing LLDP and CDP data with the following commands:
+### Empty map message
+
+{{< img src="/network_device_monitoring/network_topology_map/no_devices_found.png" alt="The no devices found message that is displayed when NDM is either not configured or because of filtering." style="width:80%;" >}}
+
+There are no devices because NDM is not configured.
+
+### No connections found / No connected devices to show
+
+{{< img src="/network_device_monitoring/network_topology_map/no_connections_found.png" alt="The no devices found message that is displayed when NDM is either not configured or because of filtering." style="width:80%;" >}}
+
+- Turn the "Hide _N_ Unconnected Devices" toggle off to show the isolated devices.
+- Use categorization tag to help understand your map view with information hierarchy.
+
+### Empty map/ no monitored devices
+
+- Ensure the "Hide _N_ Unconnected Devices toggle is off.
+
+### Missing devices/connections
+
+The Device Topology Map data is based on LLDP (Link Layer Discovery Protocol) and CDP (Cisco Discovery Protocol) information collected with SNMP. If your map is missing devices and/or connections, ensure the following:
+
+- Datadog Agent version 7.52 or later is installed.
+- Devices have LLDP and/or CDP enabled with SNMP.
+
+Verify that your devices are exposing LLDP and CDP data with the following commands:
+
+For LLDP data:
 
 ```yaml
 sudo -u dd-agent datadog-agent snmp walk <DEVICE_IP> 1.0.8802
 ```
-
-```yaml
+For CDP data
+```yaml:
 sudo -u dd-agent datadog-agent snmp walk <DEVICE_IP> 1.3.6.1.4.1.9.9.23
 ```
 
+### Missing connections or links 
+
+If your device is exposing topology data with LLDP or CDP but some of the connections are missing, ensure that the "Hide _N_ Unmonitored Devices" toggle is off.
```

Comment:
do you want to use quotes around UI elements? Typically, we bold them.

Reply:
let me ask the PM about this, thanks!